sale_order_packaging_import/demo/demo_data.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="0">
+
+    <record id="test_package" model="product.packaging">
+        <field name="name">Package EDIFACT Demo 1</field>
+        <field name="product_id" ref='product.product_delivery_02' />
+        <field name="sales" eval="True" />
+        <field name="qty">12</field>
+        <field name="barcode">PACKDEMO1</field>
+    </record>
+
+</odoo>
```

Comment:
I think this file is not needed, in `Tests`, it has been created by code for testing
If it is used for testing in EDIFACT, we can move to `sale_order_import_edifact`

Reply:
btw lgtm